migration/migrating_4_1_4/migrating-applications-with-cam.adoc

```diff
@@ -6,7 +6,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/migration-launching-cam.adoc[leveloffset=+1]
+You can migrate application workloads by adding your clusters and replication repository to the CAM web console. Then you can create and run a migration plan.
```

Comment:
"You can migrate application workloads by adding your clusters and replication repository to the CAM web console to create and run a migration plan."